front/components/assistant_builder/FolderOrWebsiteTree.tsx

```diff
@@ -0,0 +1,81 @@
+import { FolderIcon, GlobeAltIcon, Tree } from "@dust-tt/sparkle";
+import type { LightContentNode, WorkspaceType } from "@dust-tt/types";
+
+import type { DataSourceForAssistantBuilderType } from "@app/components/assistant_builder/shared";
+import type { AssistantBuilderDataSourceConfiguration } from "@app/components/assistant_builder/types";
+import DataSourceResourceSelectorTree from "@app/components/DataSourceResourceSelectorTree";
+import { useParentResourcesById } from "@app/hooks/useParentResourcesById";
+
+export default function FolderOrWebsiteTree({
```

Comment:
`nit` for small component like we are okay to have them live next to their wrapper/usage. In this case `FolderOrWebsiteResourceSelector`.